ui/hooks/useTransactionEventFragment.js

```diff
@@ -1,10 +1,28 @@
-import { useCallback } from 'react';
+import { useCallback, useEffect } from 'react';
```

Comment:
Should these changes be in a separate PR? Are these just things that got missed or things that we couldn't do while the feature flag was there?

Reply:
@mcmire : this was thing that was missed out previously. These were actually review feedbacks on this PR in fact.

Ideally yes we should have planned a different PR for this. But work is completed on this PR,  thus I would prefer to not spend time now on splitting it.

Reply:
Okay. It might be good to split this off into a separate commit at least if you can, so if someone is trying to do a `git blame` on this later they don't get confused.